src/screens/Sidebar/Sidebar.res

```diff
@@ -606,6 +616,9 @@ let make = (
           </div>
           <PinIconComponentStates isHSSidebarPinned setIsSidebarExpanded isSidebarExpanded />
         </div>
+        <RenderIf condition={featureFlagDetails.userManagementRevamp}>
```

Comment:
Can we modify the condition to also check for totp feature flag as well ? If totp is not enabled, the context used to extract the organization, merchant, and profile IDs will fail.